Formula/brev-cli.rb

```diff
@@ -0,0 +1,35 @@
+class BrevCli < Formula
+  desc "CLI tool for managing workspaces provided by brev.dev"
+  homepage "https://docs.brev.dev"
+  version "0.6.12"
+
+  on_macos do
```

Comment:
These are binary artefacts. We don't ship these in Homebrew/core. This should be built from source.

Reply:
thanks for getting back so soon, I'll fix this up